### PR TITLE
feat(react-native): Add `@sentry/wizard` to Expo guide

### PR DESCRIPTION
**Pull request by krystofwoldrich**
This PR adds Wizard to Expo Guide

Wizad supports Expo since https://github.com/getsentry/sentry-wizard/releases/tag/v3.25.0

### CODE REVIEW

File: docs/platforms/react-native/manual-setup/expo.mdx [hunk 1]
```diff
@@ -12,9 +12,17 @@ To set up the Sentry React Native SDK in your Expo project, follow the steps on
...
-## Install the Sentry SDK
+## Automatic Install
 
-Install the `@sentry/react-native` package:
+[Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project automatically, though you can follow the [Manual Install](./#manual-install) if you prefer. You only need to patch the project once. Then you can add the patched files to your version control system.
+
+```bash {tabTitle:npm}
+npx @sentry/wizard@latest -i reactNative
+```
+
```
Comment (kahest) [inline docs/platforms/react-native/manual-setup/expo.mdx:22]: does it make sense to add a `The following tasks will be performed by the Sentry Wizard` section similar to the [main getting started docs](https://docs.sentry.io/platforms/react-native/#install)?

Comment (kahest) [reply docs/platforms/react-native/manual-setup/expo.mdx:22]: also maybe add a link to the #verify-setup section

Comment (lizokm) [reply docs/platforms/react-native/manual-setup/expo.mdx:22]: +1 to that.

File: docs/platforms/react-native/manual-setup/expo.mdx [hunk 2]
```diff
@@ -12,9 +12,17 @@ To set up the Sentry React Native SDK in your Expo project, follow the steps on
 - [Migrate from `sentry-expo` to `@sentry/react-native`](/platforms/react-native/migration/sentry-expo/).
 - [Sign up for an account](https://sentry.io/signup/).
 
-## Install the Sentry SDK
+## Automatic Install
 
-Install the `@sentry/react-native` package:
+[Sentry Wizard](https://github.com/getsentry/sentry-wizard) will patch your project automatically, though you can follow the [Manual Install](./#manual-install) if you prefer. You only need to patch the project once. Then you can add the patched files to your version control system.
```
Comment (vivianyentran) [inline docs/platforms/react-native/manual-setup/expo.mdx:17]:
```suggestion
Use the [Sentry Wizard](https://github.com/getsentry/sentry-wizard) to patch your project automatically, as shown below. Alternatively, you can follow the [Manual Install](./#manual-install) if you prefer. You only need to patch the project once. Then, you can add the patched files to your version control system.
```

**Review by vivianyentran — APPROVED** [review]

File: docs/platforms/react-native/manual-setup/expo.mdx (hunk 1 above)
Comment (krystofwoldrich) [reply docs/platforms/react-native/manual-setup/expo.mdx:22]: The link to verify setup is not necessary, the wizard prompts users to verify the setup and collect information about success/failure.

Comment (krystofwoldrich) [reply docs/platforms/react-native/manual-setup/expo.mdx:22]: The list of what wizard docs totally make sense, I'll add it, thank you.

**Pull request merged**